ui/app/.huskyrc

```diff
@@ -0,0 +1,5 @@
+{
+    "hooks": {
```

Comment:
@janfuhrer Maybe you need to explain me how husky works in a sub folder of a git repo that already has hooks. And as far as I understand it the husky setup is a bit different now: https://github.com/typicode/husky

Reply:
Removed husky. Everything done we pre commit now.